frontend_addon/cookiecutter.json

```diff
@@ -6,6 +6,7 @@
   "email": "collective@plone.org",
   "github_organization": "collective",
   "npm_package_name": "{{ cookiecutter.frontend_addon_name }}",
+  "year": "{% now 'local', '%Y' %}",
```

Comment:
Please change this to use `__year` instead of `year` so that the user isn't prompted to enter a year.